tests/unit/sagemaker/jumpstart/utils.py

```diff
@@ -31,15 +32,22 @@
 
 
 def get_header_from_base_header(
-    region: str = None, model_id: str = None, version: str = None
+    _obj: JumpStartModelsCache = None,
+    region: str = None,
+    model_id: str = None,
+    semantic_version_str: str = None,
```

Comment:
are you adding this argument so that you can `get_header_from_base_header(**kwargs)` ?

If that's the case, could you please add a sanity check:
```py
if version and semantic_version_str:
    raise ValueError()
```